<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.4.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-minimal.min.css">
  <script src="/lib/pace/pace.min.js"></script>


  <script>
  (function(i,s,o,g,r,a,m){i["DaoVoiceObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;a.charset="utf-8";m.parentNode.insertBefore(a,m)})(window,document,"script",('https:' == document.location.protocol ? 'https:' : 'http:') + "//widget.daovoice.io/widget/0f81ff2f.js","daovoice")
  daovoice('init', {
      app_id: "237cb675"
    });
  daovoice('update');
  </script>


<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"lucinda315.github.io","root":"/","scheme":"Mist","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta name="description" content="InroductionBioT2Ex  provides a package or toolkit to conveniently convert the tabular txt file into a well-anotated Excel file. We believe that BioT2Ex could facilitate the downstream data interpretat">
<meta property="og:type" content="website">
<meta property="og:title" content="Using BioT2Ex">
<meta property="og:url" content="https://lucinda315.github.io/document/index.html">
<meta property="og:site_name" content="BioT2Ex">
<meta property="og:description" content="InroductionBioT2Ex  provides a package or toolkit to conveniently convert the tabular txt file into a well-anotated Excel file. We believe that BioT2Ex could facilitate the downstream data interpretat">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="c:/Users/61972/Desktop/renderedgeR.png">
<meta property="og:image" content="c:/Users/61972/Desktop/renderDESeq2.png">
<meta property="og:image" content="c:/Users/61972/Desktop/renderChIPseeker.png">
<meta property="og:image" content="c:/Users/61972/Desktop/description.png">
<meta property="og:image" content="c:/Users/61972/Desktop/effect.png">
<meta property="og:image" content="c:/Users/61972/Desktop/head.png">
<meta property="og:image" content="c:/Users/61972/Desktop/color.png">
<meta property="article:published_time" content="2021-11-27T16:18:10.763Z">
<meta property="article:modified_time" content="2021-11-27T16:16:23.042Z">
<meta property="article:author" content="BioT2Ex Group">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="c:/Users/61972/Desktop/renderedgeR.png">

<link rel="canonical" href="https://lucinda315.github.io/document/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : false,
    lang   : 'en'
  };
</script>

  <title>Using BioT2Ex | BioT2Ex
</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>
    
    <a target="_blank" rel="noopener" href="https://github.com/Lucinda315">
    <img width="149" height="149" 
     style="position: absolute; top: 0; right: 0; border: 0;" 
      src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1">
    </a>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">BioT2Ex</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>Categories</a>

  </li>
        <li class="menu-item menu-item-docs">

    <a href="/document/" rel="section"><i class="fa fa-file-alt fa-fw"></i>Docs</a>

  </li>
        <li class="menu-item menu-item-contribute">

    <a href="/contribute/" rel="section"><i class="fa fa-laptop fa-fw"></i>Contribute</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>About</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          
  
  

          <div class="content page posts-expand">
            

    
    
    
    <div class="post-block" lang="en">
      <header class="post-header">

<h1 class="post-title" itemprop="name headline">Using BioT2Ex
</h1>

<div class="post-meta">
  

</div>

</header>

      
      
      
      <div class="post-body">
          <h2 id="Inroduction"><a href="#Inroduction" class="headerlink" title="Inroduction"></a>Inroduction</h2><p>BioT2Ex  provides a package or toolkit to conveniently convert the tabular txt file into a well-anotated Excel file. We believe that BioT2Ex could facilitate the downstream data interpretation of high-throughput genomic sequencing analysis.</p>
<p>BioT2Ex now supports the results of the RNA-seq analysis,Chip-seq analysis and Wes data analysis.</p>
<h2 id="Installing-BioT2Ex"><a href="#Installing-BioT2Ex" class="headerlink" title="Installing  BioT2Ex"></a>Installing  BioT2Ex</h2><h3 id="Installing-in-R"><a href="#Installing-in-R" class="headerlink" title="Installing in R"></a>Installing in R</h3><figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">if</span> (!requireNamespace(<span class="string">&quot;BiocManager&quot;</span>, quietly = <span class="literal">TRUE</span>))</span><br><span class="line">    install.packages(<span class="string">&quot;BiocManager&quot;</span>)</span><br><span class="line"></span><br><span class="line">BiocManager::install(<span class="string">&quot;BioT2Ex&quot;</span>)</span><br></pre></td></tr></table></figure>
<h3 id="Installing-in-Python"><a href="#Installing-in-Python" class="headerlink" title="Installing in Python"></a>Installing in Python</h3><figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">pip install BioT2Ex</span><br></pre></td></tr></table></figure>

<h2 id="Running-BioT2Ex-in-R"><a href="#Running-BioT2Ex-in-R" class="headerlink" title="Running BioT2Ex in R"></a>Running BioT2Ex in R</h2><p>Once installed the package, just go to your analysis directory and use the functions.</p>
<h3 id="Render-the-output"><a href="#Render-the-output" class="headerlink" title="Render the output"></a>Render the output</h3><p>Select the appropriate render function according to the output results, and conduct a preliminary analysis and processing of your output results and format conversion. Currently BioT2Ex’s render function supports output from <strong>edgeR</strong>, <strong>DESeq2</strong> and <strong>ChIPseeker</strong>.</p>
<h4 id="renderedgeR"><a href="#renderedgeR" class="headerlink" title="renderedgeR"></a>renderedgeR</h4><p>The output results were sorted in descending order according to the size of foldchange, annotations representing up-regulation or down-regulation of expression were added, geneId was added, then GO enrichment analysis was further conducted, GO number was output, and Genebank link of each gene was added.A list of table header comments is also generated in this step.</p>
<p>The original expression difference analysis data are from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">paper</a>.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># @param input  input data. dataframe</span></span><br><span class="line"><span class="comment"># @param orgDB  the org.xx.eg.db you need. character</span></span><br><span class="line"><span class="comment"># @param log2FC Set the log2FC value to sift data</span></span><br><span class="line"><span class="comment"># @param n The number of genes you want to choose, default is 20000.character</span></span><br><span class="line"><span class="comment"># @param showInExcel show output in Excel or not</span></span><br><span class="line">renderedgeR(input,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,log2FC=<span class="number">1</span>,n=<span class="number">20000</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="C:\Users\61972\Desktop\renderedgeR.png" alt="renderedgeR"></p>
<h4 id="renderDESeq2"><a href="#renderDESeq2" class="headerlink" title="renderDESeq2"></a>renderDESeq2</h4><p>The output results were sorted in descending order according to the size of foldchange, annotations representing up-regulation or down-regulation of expression were added, geneId was added, then GO enrichment analysis was further conducted, GO number was output, and Genebank link of each gene was added. A list of table header comments is also generated in this step.</p>
<p>The original expression difference analysis data are from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">paper</a>.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># @param input  input data. dataframe</span></span><br><span class="line"><span class="comment"># @param orgDB  the org.xx.eg.db you need. character</span></span><br><span class="line"><span class="comment"># @param log2FC Set the log2FC value to sift data</span></span><br><span class="line"><span class="comment"># @param n The number of as you want to choose, default is 20000.character</span></span><br><span class="line"><span class="comment"># @param showInExcel show output in Excel or not</span></span><br><span class="line">renderDESeq2(input,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,log2FC=<span class="number">1</span>,n=<span class="number">20000</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<img src="C:\Users\61972\Desktop\renderDESeq2.png" alt="renderDESeq2" style="zoom:150%;" />

<h4 id="renderChIPseeker"><a href="#renderChIPseeker" class="headerlink" title="renderChIPseeker"></a>renderChIPseeker</h4><p>Add “ENSEMBL”,”SYMBOL”,”GENENAME” and a list of table header comments is also generated in this step.</p>
<p>The original data is downloaded from the <a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144195">GEO database</a>.(GSE144195_combined_peaks_with_CTCF_motif.bed.gz)</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># @param input input data .dataframe</span></span><br><span class="line"><span class="comment"># @param orgDB  the org.xx.eg.db you need. character</span></span><br><span class="line"><span class="comment"># @param showInExcel show output in Excel or not</span></span><br><span class="line">renderChIPseeker(input,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="C:\Users\61972\Desktop\renderChIPseeker.png" alt="renderChIPseeker"></p>
<h3 id="Add-gene-description"><a href="#Add-gene-description" class="headerlink" title="Add gene description"></a>Add gene description</h3><p>A function to add  description of genes by geneId.</p>
<h4 id="addDescription"><a href="#addDescription" class="headerlink" title="addDescription"></a>addDescription</h4><p>To use this function you need to enter the taxID of the species and the column number of geneId.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># @param input input data. dataframe</span></span><br><span class="line"><span class="comment"># @param taxid  tax_id of the organism,can query by TaxIdentifier.character</span></span><br><span class="line"><span class="comment"># @param ncol  the column number of geneId</span></span><br><span class="line"><span class="comment"># @param showInExcel show output in Excel or not</span></span><br><span class="line">addDescription(input,taxid=<span class="number">9606</span>,ncol=<span class="number">9</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<img src="C:\Users\61972\Desktop\description.png" alt="description" style="zoom:150%;" />



<h3 id="Write-the-data-into-excel"><a href="#Write-the-data-into-excel" class="headerlink" title="Write the data into excel"></a>Write the data into excel</h3><p>This function inputs the previously processed data into workbook and  name the workbook.</p>
<h4 id="writeOut"><a href="#writeOut" class="headerlink" title="writeOut"></a>writeOut</h4><figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># @param input  input data .dataframe</span></span><br><span class="line"><span class="comment"># @param package  which R package outputs the data,character</span></span><br><span class="line"><span class="comment"># @param showInExcel show output in Excel or not</span></span><br><span class="line">writeOut(input,package=<span class="string">&#x27;DESeq2&#x27;</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="C:\Users\61972\Desktop\effect.png" alt="effect"></p>
<h3 id="Add-header-comments"><a href="#Add-header-comments" class="headerlink" title="Add header comments"></a>Add header comments</h3><p>Add header comments to the output result.</p>
<h4 id="addHeadAnnotation"><a href="#addHeadAnnotation" class="headerlink" title="addHeadAnnotation"></a>addHeadAnnotation</h4><p>This function adds comment information to the header, either using default arguments or custom comment content.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># @param type the way to add head annotation,character</span></span><br><span class="line"><span class="comment"># @param workbook which workbook to add feature</span></span><br><span class="line"><span class="comment"># @param worksheet which worksheet to add feature,integer</span></span><br><span class="line"><span class="comment"># @param mycomments add the custom head annotation by yourself,character</span></span><br><span class="line"><span class="comment"># @param showInExcel show output in Excel or not</span></span><br><span class="line">addHeadAnnotation(type=<span class="string">&#x27;default&#x27;</span>,wbe1,worksheet=<span class="number">1</span>,showInExcel=<span class="literal">TRUE</span>)</span><br><span class="line">addHeadAnnotation(type=<span class="string">&quot;userdefine&quot;</span>,wb,worksheet=<span class="number">1</span>,mycomments=<span class="built_in">c</span>(<span class="string">&#x27;a&#x27;</span>,<span class="string">&#x27;b&#x27;</span>,<span class="string">&#x27;c&#x27;</span>),showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="C:\Users\61972\Desktop\head.png" alt="head"></p>
<h3 id="Add-feature-to-the-excel"><a href="#Add-feature-to-the-excel" class="headerlink" title="Add feature to the excel"></a>Add feature to the excel</h3><p>This function modifies the tabular style of the output.</p>
<h4 id="addFeature"><a href="#addFeature" class="headerlink" title="addFeature"></a>addFeature</h4><p>This function will set a uniform font size for the output results, and add a gradient representing the expression level and size for the foldchange column. Red represents up and blue represents down.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span> input data .dataframe</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">workbook</span></span> which workbook to add feature</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">worksheet</span></span> which worksheet to add feature,integer</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">colorcol</span></span> the column number of log2foldchange ,numerical</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> show output in Excel or not</span></span><br><span class="line">addFeature(input,workbook,worksheet = <span class="number">1</span>,colorcol = <span class="number">3</span>,showInExcel = <span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="C:\Users\61972\Desktop\color.png" alt="color"></p>
<h3 id="Other-functions"><a href="#Other-functions" class="headerlink" title="Other functions"></a>Other functions</h3><p>These functions are included in some main function functions to further improve their effects.</p>
<h4 id="correctNumber"><a href="#correctNumber" class="headerlink" title="correctNumber"></a>correctNumber</h4><p>This function can select the numeric column in the file and makes it displays correctly.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">as.data.frame(a)</span><br><span class="line">correctNumber(a$<span class="number">2</span>)</span><br></pre></td></tr></table></figure>

<h4 id="showInExcel"><a href="#showInExcel" class="headerlink" title="showInExcel"></a>showInExcel</h4><p>A function to open selected directory or file in excel, this base fuction is copied from o function of yulab.utils(package).</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">showInExcel(a,<span class="string">&quot;mydata.xlsx&quot;</span>)</span><br></pre></td></tr></table></figure>

<h4 id="renderPackage"><a href="#renderPackage" class="headerlink" title="renderPackage"></a>renderPackage</h4><p>This is a function integration function, which can directly complete the above analysis steps. Currently, only edger, deseq2 and chipseeker output results are supported.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">renderPackage(chip,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,package =<span class="string">&quot;ChIPseeker&quot;</span>,taxid=<span class="number">9606</span>,showInExcel=<span class="literal">TRUE</span>)</span><br><span class="line">renderPackage(y,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,package =<span class="string">&quot;edgeR&quot;</span>,taxid=<span class="number">9606</span>,log2FC=<span class="number">1</span>,n=<span class="number">20000</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>



<h2 id="Running-BioT2Ex-in-Python"><a href="#Running-BioT2Ex-in-Python" class="headerlink" title="Running BioT2Ex in Python"></a>Running BioT2Ex in Python</h2>
      </div>
      
      
      
    </div>
    

    
    
    


          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Inroduction"><span class="nav-number">1.</span> <span class="nav-text">Inroduction</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Installing-BioT2Ex"><span class="nav-number">2.</span> <span class="nav-text">Installing  BioT2Ex</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Installing-in-R"><span class="nav-number">2.1.</span> <span class="nav-text">Installing in R</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Installing-in-Python"><span class="nav-number">2.2.</span> <span class="nav-text">Installing in Python</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Running-BioT2Ex-in-R"><span class="nav-number">3.</span> <span class="nav-text">Running BioT2Ex in R</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Render-the-output"><span class="nav-number">3.1.</span> <span class="nav-text">Render the output</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#renderedgeR"><span class="nav-number">3.1.1.</span> <span class="nav-text">renderedgeR</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#renderDESeq2"><span class="nav-number">3.1.2.</span> <span class="nav-text">renderDESeq2</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#renderChIPseeker"><span class="nav-number">3.1.3.</span> <span class="nav-text">renderChIPseeker</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Add-gene-description"><span class="nav-number">3.2.</span> <span class="nav-text">Add gene description</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#addDescription"><span class="nav-number">3.2.1.</span> <span class="nav-text">addDescription</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Write-the-data-into-excel"><span class="nav-number">3.3.</span> <span class="nav-text">Write the data into excel</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#writeOut"><span class="nav-number">3.3.1.</span> <span class="nav-text">writeOut</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Add-header-comments"><span class="nav-number">3.4.</span> <span class="nav-text">Add header comments</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#addHeadAnnotation"><span class="nav-number">3.4.1.</span> <span class="nav-text">addHeadAnnotation</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Add-feature-to-the-excel"><span class="nav-number">3.5.</span> <span class="nav-text">Add feature to the excel</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#addFeature"><span class="nav-number">3.5.1.</span> <span class="nav-text">addFeature</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Other-functions"><span class="nav-number">3.6.</span> <span class="nav-text">Other functions</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#correctNumber"><span class="nav-number">3.6.1.</span> <span class="nav-text">correctNumber</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#showInExcel"><span class="nav-number">3.6.2.</span> <span class="nav-text">showInExcel</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#renderPackage"><span class="nav-number">3.6.3.</span> <span class="nav-text">renderPackage</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Running-BioT2Ex-in-Python"><span class="nav-number">4.</span> <span class="nav-text">Running BioT2Ex in Python</span></a></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">BioT2Ex Group</p>
  <div class="site-description" itemprop="description">BioT2Ex project provides a package or toolkit to conveniently render the tabular txt file into well-anotated Excel files</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives">
          <span class="site-state-item-count">1</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
  </nav>
</div>
  <div class="sidebar-button motion-element"><i class="fa fa-comment"></i>
    Chat
  </a>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2021</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">BioT2Ex Group</span>
</div>

<div class="powered-by">
<i class="fa fa-user-md"></i><span id="busuanzi_container_site_uv">
  Site visits:<span id="busuanzi_value_site_uv"></span>
</span>
</div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>


  <script defer src="/lib/three/three.min.js"></script>
    <script defer src="/lib/three/canvas_lines.min.js"></script>


  















  

  

</body>
</html>
