<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.4.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-minimal.min.css">
  <script src="/lib/pace/pace.min.js"></script>



<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"lucinda315.github.io","root":"/","scheme":"Mist","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta name="description" content="IntroductionBioT2Ex  provides a package or toolkit to conveniently convert the tabular TXT file into a well-annotated Excel file. BioT2Ex could facilitate the downstream data interpretation of high-th">
<meta property="og:type" content="website">
<meta property="og:title" content="Using BioT2Ex">
<meta property="og:url" content="https://lucinda315.github.io/document/index.html">
<meta property="og:site_name" content="BioT2Ex">
<meta property="og:description" content="IntroductionBioT2Ex  provides a package or toolkit to conveniently convert the tabular TXT file into a well-annotated Excel file. BioT2Ex could facilitate the downstream data interpretation of high-th">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="https://lucinda315.github.io/document/DGEExact.png">
<meta property="og:image" content="https://lucinda315.github.io/document/edgerdata.png">
<meta property="og:image" content="https://lucinda315.github.io/document/edgeR.gif">
<meta property="og:image" content="https://lucinda315.github.io/document/updown.png">
<meta property="og:image" content="https://lucinda315.github.io/document/DESeqDataSet.png">
<meta property="og:image" content="https://lucinda315.github.io/document/deseqdata.png">
<meta property="og:image" content="https://lucinda315.github.io/document/deseq2.gif">
<meta property="og:image" content="https://lucinda315.github.io/document/updown_deseq.png">
<meta property="og:image" content="https://lucinda315.github.io/document/limma.png">
<meta property="og:image" content="https://lucinda315.github.io/document/render_limma.png">
<meta property="og:image" content="https://lucinda315.github.io/document/updown_limma.png">
<meta property="og:image" content="https://lucinda315.github.io/document/enrichresultgo.png">
<meta property="og:image" content="https://lucinda315.github.io/document/go.png">
<meta property="og:image" content="https://lucinda315.github.io/document/goresult.png">
<meta property="og:image" content="https://lucinda315.github.io/document/goresult2.png">
<meta property="og:image" content="https://lucinda315.github.io/document/enrichresultkegg.png">
<meta property="og:image" content="https://lucinda315.github.io/document/kegg.png">
<meta property="og:image" content="https://lucinda315.github.io/document/KEGGresult.png">
<meta property="og:image" content="https://lucinda315.github.io/document/chipseeker.gif">
<meta property="og:image" content="https://lucinda315.github.io/document/description.png">
<meta property="og:image" content="https://lucinda315.github.io/document/effect.png">
<meta property="og:image" content="https://lucinda315.github.io/document/head.png">
<meta property="og:image" content="https://lucinda315.github.io/document/color.png">
<meta property="og:image" content="https://lucinda315.github.io/document/macs.png">
<meta property="og:image" content="https://lucinda315.github.io/document/MutSigCV.png">
<meta property="og:image" content="https://lucinda315.github.io/document/RSEM.png">
<meta property="article:published_time" content="2022-02-25T13:09:45.185Z">
<meta property="article:modified_time" content="2022-02-25T13:09:20.512Z">
<meta property="article:author" content="BioT2Ex Group">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://lucinda315.github.io/document/DGEExact.png">

<link rel="canonical" href="https://lucinda315.github.io/document/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : false,
    lang   : 'en'
  };
</script>

  <title>Using BioT2Ex | BioT2Ex
</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>
    
    <a target="_blank" rel="noopener" href="https://github.com/Lucinda315">
    <img width="149" height="149" 
     style="position: absolute; top: 0; right: 0; border: 0;" 
      src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1">
    </a>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">BioT2Ex</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a>

  </li>
        <li class="menu-item menu-item-tools">

    <a href="/tools/" rel="section"><i class="fa fa-tools fa-fw"></i>Supported tools</a>

  </li>
        <li class="menu-item menu-item-docs">

    <a href="/document/" rel="section"><i class="fa fa-file-alt fa-fw"></i>Docs</a>

  </li>
        <li class="menu-item menu-item-contribute">

    <a href="/contribute/" rel="section"><i class="fa fa-laptop fa-fw"></i>Contribute</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>About</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          
  
  

          <div class="content page posts-expand">
            

    
    
    
    <div class="post-block" lang="en">
      <header class="post-header">

<h1 class="post-title" itemprop="name headline">Using BioT2Ex
</h1>

<div class="post-meta">
  

</div>

</header>

      
      
      
      <div class="post-body">
          <h2 id="Introduction"><a href="#Introduction" class="headerlink" title="Introduction"></a>Introduction</h2><p>BioT2Ex  provides a package or toolkit to conveniently convert the tabular TXT file into a well-annotated Excel file. BioT2Ex could facilitate the downstream data interpretation of high-throughput genomic sequencing analysis.</p>
<p>BioT2Ex now supports the results of the <strong>RNA-seq</strong> analysis,<strong>ChIP-seq</strong> analysis and <strong>WES</strong> analysis.</p>
<h2 id="Installing-BioT2Ex"><a href="#Installing-BioT2Ex" class="headerlink" title="Installing  BioT2Ex"></a>Installing  BioT2Ex</h2><h3 id="Installing-in-R"><a href="#Installing-in-R" class="headerlink" title="Installing in R"></a>Installing in R</h3><figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">if</span> (!requireNamespace(<span class="string">&quot;BiocManager&quot;</span>, quietly = <span class="literal">TRUE</span>))</span><br><span class="line">    install.packages(<span class="string">&quot;BiocManager&quot;</span>)</span><br><span class="line"></span><br><span class="line">BiocManager::install(<span class="string">&quot;BioT2Ex&quot;</span>)</span><br></pre></td></tr></table></figure>
<h3 id="Installing-in-Python"><a href="#Installing-in-Python" class="headerlink" title="Installing in Python"></a>Installing in Python</h3><figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">pip install BioT2Ex</span><br></pre></td></tr></table></figure>

<h2 id="Running-BioT2Ex-in-R"><a href="#Running-BioT2Ex-in-R" class="headerlink" title="Running BioT2Ex in R"></a>Running BioT2Ex in R</h2><p>Once installed the package, just go to your analysis directory, select the appropriate rendering functions according to the output results, and perform formatting conversion and style modification as desired.</p>
<h3 id="Rendering-the-RNA-seq-analysis-results"><a href="#Rendering-the-RNA-seq-analysis-results" class="headerlink" title="Rendering the RNA-seq analysis results"></a>Rendering the RNA-seq analysis results</h3><p>Currently BioT2Ex’s render functions support the RNA-seq analysis results exported from <strong>edgeR</strong> ,  <strong>DESeq2</strong> , <strong>limma</strong> and <strong>clusterProfiler</strong>.</p>
<h4 id="render-edgeR"><a href="#render-edgeR" class="headerlink" title="render_edgeR"></a><a href="#render_edger">render_edgeR</a></h4><ol>
<li>Sort the output results descendingly based on the size of logFC (log2 transformed fold-change); add the “UP”- or “DOWN”-regulation annotations(abs(logFC)&gt;1 &amp; PValue&lt;0.05);</li>
<li> Add ENTREZID; </li>
<li> Add GO number and GeneBank link of each gene;</li>
<li> Convert the output to a file in XLSX format, add comment information for the header, and render the default table style, such as font, size, and column width;</li>
<li>Add gradient color to the logFC column to indicate the expression level, with red representing up-regulation and blue representing down-regulation;</li>
<li>Add statistical functions for some specific variables, and the statistical results are presented in additional worksheets, including the corresponding data results, and the visualized results are displayed in pie charts. </li>
</ol>
<p>The original  differential expression analysis was derived from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">publication</a>.</p>
<p><strong>Procedures:</strong></p>
<ol>
<li><p>Provide the data you want to process as input, such as DGEExact(egdeR object) or imported external TXT/CSV/Excel files as data frame and the rownames should be gene symbol ;</p>
<p><strong>For example:</strong></p>
<p><img src="DGEExact.png" alt="DGEExact"></p>
<p><strong>or:</strong></p>
<p><img src="edgerdata.png" alt="edgerdata"></p>
</li>
<li><p>Provide the OrgDb (e.g., org.Hs.eg.db for human) for gene annotation, the corresponding taxonomy ID (e.g., taxID 9606 for human and the number of genes you want to output.</p>
</li>
</ol>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data. [dataframe or DGEExact]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">orgDB</span></span>  the org.xx.eg.db you need. [character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">taxid</span></span>  tax_id of the organism,can query by TaxIdentifier.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">filename</span></span> the name of the output file.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">n</span></span> The number of as you want to choose, default is 20000.[integer]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> open the output file in Excel or not.[logical]</span></span><br><span class="line">render_edgeR(input,orgDB,taxid,n,filename,showInExcel)</span><br><span class="line">data&lt;-render_edgeR(y,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,taxid=<span class="number">9606</span>,n=<span class="number">20000</span>,filename=<span class="string">&quot;test&quot;</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p><strong>Output effect:</strong></p>
<ol>
<li>The first worksheet(edgeR-3.34.0) : The rendered differential expression analysis data:</li>
</ol>
<p><img src="edgeR.gif" alt="edgeR"></p>
<ol start="2">
<li><p>The second worksheet(up_down) : The number of up-regulated , down-regulated expression and no difference were classified and statistically displayed in a pie chart. The pie chart will also be saved locally:</p>
<img src="updown.png" alt="updown" style="zoom:80%;" /></li>
</ol>
<h4 id="render-DESeq2"><a href="#render-DESeq2" class="headerlink" title="render_DESeq2"></a><a href="#render_deseq2">render_DESeq2</a></h4><ol>
<li>Sort the output results descendingly based on the size of log2FoldChange (log2 transformed fold-change); add the “UP”- or “DOWN”-regulation annotations (abs(log2FoldChange)&gt;1 &amp; Padj&lt;0.05);</li>
<li> Add ENTREZID; </li>
<li> Add GO number and GeneBank link of each gene;</li>
<li> Convert the output to a file in XLSX format, add comment information for the header, and render the default table style, such as font, size, and column width;</li>
<li>Add gradient color to the logFC column to indicate the expression level, with red representing up-regulation and blue representing down-regulation;</li>
<li>Add statistical functions for some specific variables, and the statistical results are presented in additional worksheets, including the corresponding data results, and the visualized results are displayed in pie charts. </li>
</ol>
<p>Compared with the render result of edgeR, the  header comments will be different.The original  differential expression analysis was derived from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">publication</a>. </p>
<p><strong>Procedures:</strong></p>
<ol>
<li><p>Provide the data you want to process as input, such as DESeqDataSet(DESeq2 object) or imported external TXT/CSV/Excel files as data frame and the rownames should be gene symbol ;</p>
<p><strong>For example:</strong></p>
<p><img src="DESeqDataSet.png" alt="DESeqDataSet"></p>
<p><strong>or:</strong></p>
<p><img src="deseqdata.png" alt="deseqdata"></p>
</li>
<li><p>Provide the OrgDb (e.g., org.Hs.eg.db for human) for gene annotation, the corresponding taxonomy ID (e.g., taxID 9606 for human) and the number of genes you want to output.</p>
</li>
</ol>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data. [dataframe or DGEExact]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">orgDB</span></span>  the org.xx.eg.db you need. [character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">taxid</span></span>  tax_id of the organism,can query by TaxIdentifier.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">filename</span></span> the name of the output file.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">n</span></span> The number of as you want to choose, default is 20000.[integer]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> open the output file in Excel or not.[logical]</span></span><br><span class="line">render_DESeq2(input,orgDB,taxid,filename,n,showInExcel)</span><br><span class="line">data&lt;-render_DESeq2(deseq,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,taxid=<span class="number">9606</span>,filename=<span class="string">&quot;test_des&quot;</span>,n=<span class="number">20000</span>,showInExcel=<span class="literal">FALSE</span>)</span><br></pre></td></tr></table></figure>

<p><strong>Output effect:</strong></p>
<ol>
<li>The first worksheet(DESeq2-1.32.0) : The rendered differential expression analysis data:</li>
</ol>
<p><img src="deseq2.gif" alt="deseq2"></p>
<ol start="2">
<li><p>The second worksheet(up_down) : The number of up-regulated , down-regulated expression and no difference were classified and statistically displayed in a pie chart. The pie chart will also be saved locally:</p>
<img src="updown_deseq.png" alt="updown_deseq" style="zoom:80%;" /></li>
</ol>
<h4 id="render-limma"><a href="#render-limma" class="headerlink" title="render_limma"></a><a href="#render_limma">render_limma</a></h4><ol>
<li>Sort the output results descendingly based on the size of logFC (log2 transformed fold-change); add the “UP”- or “DOWN”-regulation annotations (abs(logFC)&gt;1 &amp; adj.P.Val&lt;0.05);</li>
<li> Add ENTREZID; </li>
<li> Add GO number and GeneBank link of each gene;</li>
<li> Convert the output to a file in XLSX format, add comment information for the header, and render the default table style, such as font, size, and column width;</li>
<li>Add gradient color to the logFC column to indicate the expression level, with red representing up-regulation and blue representing down-regulation;</li>
<li>Add statistical functions for some specific variables, and the statistical results are presented in additional worksheets, including the corresponding data results, and the visualized results are displayed in pie charts. </li>
</ol>
<p>Compared with the render result of edgeR, the  header comments will be different.The original  differential expression analysis was derived from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">publication</a>. </p>
<p><strong>Procedures:</strong></p>
<ol>
<li>Provide the data you want to process as input, such as imported external TXT/CSV/Excel files as data frame and the rownames should be gene symbol .</li>
</ol>
<p>​     <strong>For example:</strong></p>
<img src="limma.png" alt="limma" style="zoom:80%;" />

<ol start="2">
<li>Provide the OrgDb (e.g., org.Hs.eg.db for human) for gene annotation, the corresponding taxonomy ID (e.g., taxID 9606 for human) and the number of genes you want to output.</li>
</ol>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data. [dataframe or DGEExact]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">orgDB</span></span>  the org.xx.eg.db you need. [character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">taxid</span></span>  tax_id of the organism,can query by TaxIdentifier.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">filename</span></span> the name of the output file.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">n</span></span> The number of as you want to choose, default is 20000.[integer]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> open the output file in Excel or not.[logical]</span></span><br><span class="line">render_limma(input,orgDB,taxid,filename,n,showInExcel)</span><br><span class="line">data&lt;-render_limma(input,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,taxid=<span class="number">9606</span>,filename=<span class="string">&quot;test_limma&quot;</span>,n=<span class="number">20000</span>,showInExcel=<span class="literal">FALSE</span>)</span><br></pre></td></tr></table></figure>

<p><strong>Output effect:</strong></p>
<ol>
<li><p>The first worksheet(limma-3.48.1) : The rendered differential expression analysis data:</p>
<img src="render_limma.png" alt="render_limma" style="zoom:80%;" /></li>
<li><p>The second worksheet(up_down) : The number of up-regulated , down-regulated expression and no difference were classified and statistically displayed in a pie chart. The pie chart will also be saved locally:</p>
<img src="updown_limma.png" alt="updown_limma" style="zoom:80%;" /></li>
</ol>
<h4 id="render-clusterProfiler"><a href="#render-clusterProfiler" class="headerlink" title="render_clusterProfiler"></a><a href="#render_clusterprofiler">render_clusterProfiler</a></h4><p>BioT2Ex now supports the results of the clusterProfiler <strong>GO analysis</strong> ,<strong>KEGG analysis</strong> and <strong>GSEA analysis</strong>(GSEA,gseGO and gseKEGG).</p>
<h5 id="render-clusterP-GO"><a href="#render-clusterP-GO" class="headerlink" title="render_clusterP_GO"></a>render_clusterP_GO</h5><ol>
<li>Add the geneontology database link for each term;</li>
<li>Convert the output to a file in XLSX format, add comment information for the header, and render the default table style, such as font, size, and column width;</li>
<li>Add statistical functions for some specific variables, and the statistical results are presented in additional worksheets, including the corresponding data results, and the visualized results are displayed in pie charts. </li>
</ol>
<p>The original  differential expression analysis was derived from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">publication</a>. </p>
<p><strong>Procedures:</strong></p>
<ol>
<li><p>Provide the data you want to process as input, such as Large enrichResult or imported external TXT/CSV/Excel files as data frame and the rownames should be gene symbol ;</p>
<p><strong>For example:</strong></p>
<img src="enrichresultgo.png" alt="enrichresultgo" style="zoom: 80%;" />

<p><strong>or:</strong></p>
<img src="go.png" alt="go" style="zoom:80%;" />

<p>2.Provide the filename you want to set.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data. [dataframe or DGEExact]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">filename</span></span> the name of the output file.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> open the output file in Excel or not.[logical]</span></span><br><span class="line">render_clusterP_GO(input,filename,showInExcel)</span><br><span class="line">data&lt;-render_clusterP_GO(input,filename=<span class="string">&quot;test&quot;</span>,showInExcel=<span class="literal">FALSE</span>)</span><br></pre></td></tr></table></figure></li>
</ol>
<p><strong>Output effect:</strong></p>
<ol>
<li>The first worksheet(clusterProfiler-4.2.0) : The rendered differential expression analysis data:</li>
</ol>
<img src="goresult.png" alt="goresult" style="zoom:80%;" />

<p>2.The second worksheet(ONTOLOGY) : The number of BP , CC and MF were classified and statistically displayed in a pie chart. The pie chart will also be saved locally:</p>
<img src="goresult2.png" alt="goresult2" style="zoom:80%;" />


<h5 id="render-clusterP-KEGG"><a href="#render-clusterP-KEGG" class="headerlink" title="render_clusterP_KEGG"></a>render_clusterP_KEGG</h5><ol>
<li>Add the KEGG database link for each term;</li>
<li>Convert the output to a file in XLSX format, add comment information for the header, and render the default table style, such as font, size, and column width.</li>
</ol>
<p>The original  differential expression analysis was derived from the <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009317">publication</a>. </p>
<p><strong>Procedures:</strong></p>
<ol>
<li><p>Provide the data you want to process as input, such as Large enrichResult or imported external TXT/CSV/Excel files as data frame and the rownames should be gene symbol ;</p>
<p><strong>For example:</strong></p>
</li>
</ol>
<img src="enrichresultkegg.png" alt="enrichresultkegg" style="zoom:80%;" />

<p>  <strong>or:</strong></p>
<img src="kegg.png" alt="kegg" style="zoom:80%;" />

<p>2.Provide the filename you want to set.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data. [dataframe or DGEExact]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">filename</span></span> the name of the output file.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> open the output file in Excel or not.[logical]</span></span><br><span class="line">render_clusterP_KEGG(input,filename,showInExcel)</span><br><span class="line">data&lt;-render_clusterP_KEGG(input,filename=<span class="string">&quot;test&quot;</span>,showInExcel=<span class="literal">FALSE</span>)</span><br></pre></td></tr></table></figure>

<p><strong>Output effect</strong>:</p>
<p>The first worksheet(clusterProfiler-4.2.0) : The rendered differential expression analysis data:</p>
<img src="KEGGresult.png" alt="KEGGresult" style="zoom:80%;" />




<h3 id="Rendering-the-ChIP-seq-analysis-results"><a href="#Rendering-the-ChIP-seq-analysis-results" class="headerlink" title="Rendering the ChIP-seq analysis results"></a>Rendering the ChIP-seq analysis results</h3><p>Currently BioT2Ex supports the ChIP-seq analysis results from <strong>ChIPseeker</strong>.</p>
<h4 id="render-ChIPseeker"><a href="#render-ChIPseeker" class="headerlink" title=" render_ChIPseeker"></a><a href="#render_chipseeker"> render_ChIPseeker</a></h4><ol>
<li>Output the results in XLSX format;</li>
<li>Add corresponding “Ensembl”, “Symbol”, “GeneName” and description of gene;</li>
<li>Render the style of the XLSX output and add header comments.</li>
</ol>
<p>The original data is downloaded from the <a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144195">GEO database</a>.(GSE144195_combined_peaks_with_CTCF_motif.bed.gz).</p>
<p><strong>Procedures:</strong></p>
<ol>
<li>Provide the data you want to process as input;</li>
<li>Provide the OrgDb (e.g., org.Hs.eg.db for human) for gene annotation and the corresponding taxonomy ID (e.g., taxID 9606 for human) .</li>
</ol>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data. [dataframe or DGEExact]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">orgDB</span></span>  the org.xx.eg.db you need. [character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">taxid</span></span>  tax_id of the organism,can query by TaxIdentifier.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">filename</span></span> the name of the output file.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">op</span></span> open the output file in Excel or not.[logical]</span></span><br><span class="line">render_ChIPseeker(input,orgDB,taxid,filename,op)</span><br><span class="line">data&lt;-render_ChIPseeker(y,orgDB=<span class="string">&#x27;org.Hs.eg.db&#x27;</span>,taxid=<span class="number">9606</span>,filename=<span class="string">&quot;test&quot;</span>,op=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="chipseeker.gif" alt="chipseeker"></p>
<h3 id="Basic-functions-of-BioT2Ex-in-R"><a href="#Basic-functions-of-BioT2Ex-in-R" class="headerlink" title="Basic functions of BioT2Ex in R"></a>Basic functions of BioT2Ex in R</h3><p>We also include some functions to further improve the readability of XLSX files. </p>
<h4 id="add-Description"><a href="#add-Description" class="headerlink" title="add_Description"></a>add_Description</h4><p>A function to add description of genes based on EntrezID. You need to provide the TaxID of the species and the column number of EntrezID (col.no).</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span> input data. [dataframe]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">taxid</span></span>  tax_id of the organism,can query by TaxIdentifier.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">ncol</span></span>  the column number of geneId.[integer]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> show output in Excel or not.[logical]</span></span><br><span class="line">add_Descriptioninput,taxid,ncol,showInExcel)</span><br><span class="line">data_c1&lt;-add_Description(data_c,taxid=<span class="number">9606</span>,ncol=<span class="number">9</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="description.png" alt="description"></p>
<h4 id="write-Out"><a href="#write-Out" class="headerlink" title="write_Out"></a>write_Out</h4><p>This function appends the previously processed data into a workbook and names the workbook.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span>  input data .[dataframe]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">package</span></span>  which R package outputs the data.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> show output in Excel or not.[logical]</span></span><br><span class="line">write_Out(input,package,showInExcel)</span><br><span class="line">wbc1&lt;-write_Out(data_c1,package=<span class="string">&#x27;ChIPseeker&#x27;</span>,showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="effect.png" alt="effect"></p>
<h4 id="add-HeadAnnotation"><a href="#add-HeadAnnotation" class="headerlink" title="add_HeadAnnotation"></a>add_HeadAnnotation</h4><p>Add header comments to the output result. Users can select the default line header comments (currently only the output results of edgeR, DESeq2 and ChIPseeker are supported) or customize the list of line header comments.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">type</span></span> the way to add head annotation.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">workbook</span></span> which workbook to add feature</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">worksheet</span></span> which worksheet to add feature.[integer]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">mycomments</span></span> add the custom head annotation by yourself.[character]</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> show output in Excel or not.[logical]</span></span><br><span class="line">add_HeadAnnotation(type,workbook,worksheet,mycomments,showInExcel)</span><br><span class="line">add_HeadAnnotation(type=<span class="string">&#x27;default&#x27;</span>,wbe1,worksheet=<span class="number">1</span>,showInExcel=<span class="literal">TRUE</span>)</span><br><span class="line">add_HeadAnnotation(type=<span class="string">&quot;userdefine&quot;</span>,wb,worksheet=<span class="number">1</span>,mycomments=<span class="built_in">c</span>(<span class="string">&#x27;a&#x27;</span>,<span class="string">&#x27;b&#x27;</span>,<span class="string">&#x27;c&#x27;</span>),showInExcel=<span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="head.png" alt="head"></p>
<h4 id="add-Feature"><a href="#add-Feature" class="headerlink" title="add_Feature"></a>add_Feature</h4><p>This function modifies the tabular style of the output.This function will set a uniform font size for the output results, and add a gradient representing the expression level and size for the foldchange column. Red represents up and blue represents down.Users need to enter the column number of log2foldchange.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">input</span></span> input data .dataframe</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">workbook</span></span> which workbook to add feature</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">worksheet</span></span> which worksheet to add feature,integer</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">colorcol</span></span> the column number of log2foldchange ,numerical</span></span><br><span class="line"><span class="comment">#&#x27; <span class="doctag">@param <span class="variable">showInExcel</span></span> show output in Excel or not</span></span><br><span class="line">add_Feature(input,workbook,worksheet,colorcol,showInExcel)</span><br><span class="line">wbe3 &lt;- add_Feature(data_e1,wbe2,worksheet = <span class="number">1</span>,colorcol = <span class="number">2</span>,showInExcel = <span class="literal">TRUE</span>)</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="color.png" alt="color"></p>
<h4 id="fix-Number"><a href="#fix-Number" class="headerlink" title="fix_Number"></a>fix_Number</h4><p>This function can select the numeric column in the file and makes it displays correctly.</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">as.data.frame(a)</span><br><span class="line">fix_Number(a$<span class="number">2</span>)</span><br></pre></td></tr></table></figure>

<h4 id="showInExcel"><a href="#showInExcel" class="headerlink" title="showInExcel"></a>showInExcel</h4><p>A function to open selected directory or file in excel, this base fuction is copied from o function of yulab.utils(package).</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">showInExcel(a,<span class="string">&quot;mydata.xlsx&quot;</span>)</span><br></pre></td></tr></table></figure>

<h4 id="op"><a href="#op" class="headerlink" title="op"></a>op</h4><p>A function to  open selected files (in Excel) automatically. This fuction is copied from o function of yulab.utils</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">op(file=<span class="string">&#x27;myfile&#x27;</span>)</span><br></pre></td></tr></table></figure>



<h2 id="Running-BioT2Ex-in-Python"><a href="#Running-BioT2Ex-in-Python" class="headerlink" title="Running BioT2Ex in Python"></a>Running BioT2Ex in Python</h2><p>BioT2Ex now supports the results of the <strong>MACS</strong>,<strong>MutSigCV</strong>,<strong>RSEM</strong> and <strong>ANNOVAR</strong>.</p>
<h3 id="Annotate-the-analysis-result-of-MACS"><a href="#Annotate-the-analysis-result-of-MACS" class="headerlink" title="Annotate the analysis result of MACS"></a>Annotate the analysis result of MACS</h3><p>The data used for the example is from xx</p>
<h4 id="BioT2Ex-m-macs"><a href="#BioT2Ex-m-macs" class="headerlink" title="BioT2Ex -m macs"></a><a href="#biot2ex -m macs">BioT2Ex -m macs</a></h4><p>Using this function will render the input data as follows: output the pre-comment and post-comment data in XLSX format, add gradients to the -log10(pvalue) column to represent the numerical size, and add header comments to the output table, beautifying the table font, alignment and formatting, etc.</p>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Python BioT2Ex -m macs [path of the inputfile]</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="macs.png" alt="macs"></p>
<h3 id="Annotate-the-analysis-result-of-MutSigCV"><a href="#Annotate-the-analysis-result-of-MutSigCV" class="headerlink" title="Annotate the analysis result of MutSigCV"></a>Annotate the analysis result of MutSigCV</h3><p>The data used for the example is from this <a target="_blank" rel="noopener" href="https://software.broadinstitute.org/cancer/cga/mutsig_run">site</a>.</p>
<h4 id="BioT2Ex-m-mutsigcv"><a href="#BioT2Ex-m-mutsigcv" class="headerlink" title="BioT2Ex -m mutsigcv"></a><a href="#biot2ex -m mutsigcv">BioT2Ex -m mutsigcv</a></h4><p>Using this function to convert input data to XLSX format, add header comments, and render table style.</p>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Python BioT2Ex -m mutsigcv [path of the inputfile]</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<img src="MutSigCV.png" alt="MutSigCV" style="zoom:150%;" />

<h3 id="Annotate-the-analysis-result-of-RSEM"><a href="#Annotate-the-analysis-result-of-RSEM" class="headerlink" title="Annotate the analysis result of RSEM"></a>Annotate the analysis result of RSEM</h3><p>The data used for the example is from xx</p>
<h4 id="BioT2Ex-m-rsem"><a href="#BioT2Ex-m-rsem" class="headerlink" title="BioT2Ex -m rsem"></a><a href="#biot2ex -m rsem">BioT2Ex -m rsem</a></h4><p>Using this function to convert input data to XLSX format, add header comments, and render table style.</p>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Python BioT2Ex -m rsem [path of the inputfile]</span><br></pre></td></tr></table></figure>

<p>Output effect:</p>
<p><img src="RSEM.png" alt="RSEM"></p>

      </div>
      
      
      
    </div>
    

    
    
    


          </div>
          
    <div class="comments" id="gitalk-container"></div>


<div id="gitment_title" class="gitment_title"></div>
<div id="container" style="display:none"></div>
<link rel="stylesheet" href="https://jjeejj.github.io/css/gitment.css">
<script src="https://jjeejj.github.io/js/gitment.js"></script>
<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
<script>
  const myTheme = {
    render(state, instance) {
      const container = document.createElement('div');
      container.lang = "en-US";
      container.className = 'gitment-container gitment-root-container';
      container.appendChild(instance.renderHeader(state, instance));
      container.appendChild(instance.renderEditor(state, instance));
      container.appendChild(instance.renderComments(state, instance));
      container.appendChild(instance.renderFooter(state, instance));
      return container;
    }
  }
 
  function showGitment() {
    $("#gitment_title").attr("style", "display:none");
    $("#container").attr("style", "").addClass("gitment_container");
    var gitment = new Gitment({
      id: '<%= page.title %>',
      theme: myTheme,
      owner: 'Lucinda315',
      repo: 'Lucinda315.github.io',
      oauth: {
        client_id: '96ebdb403b86b88077fb',
        client_secret: 'de774b01ddd6f1f1e5fc1f72d60f1a294061d4d5'
      }
    });
    gitment.render('container');
  }
 
  showGitment();
</script>



<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Introduction"><span class="nav-number">1.</span> <span class="nav-text">Introduction</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Installing-BioT2Ex"><span class="nav-number">2.</span> <span class="nav-text">Installing  BioT2Ex</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Installing-in-R"><span class="nav-number">2.1.</span> <span class="nav-text">Installing in R</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Installing-in-Python"><span class="nav-number">2.2.</span> <span class="nav-text">Installing in Python</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Running-BioT2Ex-in-R"><span class="nav-number">3.</span> <span class="nav-text">Running BioT2Ex in R</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Rendering-the-RNA-seq-analysis-results"><span class="nav-number">3.1.</span> <span class="nav-text">Rendering the RNA-seq analysis results</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#render-edgeR"><span class="nav-number">3.1.1.</span> <span class="nav-text">render_edgeR</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#render-DESeq2"><span class="nav-number">3.1.2.</span> <span class="nav-text">render_DESeq2</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#render-limma"><span class="nav-number">3.1.3.</span> <span class="nav-text">render_limma</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#render-clusterProfiler"><span class="nav-number">3.1.4.</span> <span class="nav-text">render_clusterProfiler</span></a><ol class="nav-child"><li class="nav-item nav-level-5"><a class="nav-link" href="#render-clusterP-GO"><span class="nav-number">3.1.4.1.</span> <span class="nav-text">render_clusterP_GO</span></a></li><li class="nav-item nav-level-5"><a class="nav-link" href="#render-clusterP-KEGG"><span class="nav-number">3.1.4.2.</span> <span class="nav-text">render_clusterP_KEGG</span></a></li></ol></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Rendering-the-ChIP-seq-analysis-results"><span class="nav-number">3.2.</span> <span class="nav-text">Rendering the ChIP-seq analysis results</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#render-ChIPseeker"><span class="nav-number">3.2.1.</span> <span class="nav-text"> render_ChIPseeker</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Basic-functions-of-BioT2Ex-in-R"><span class="nav-number">3.3.</span> <span class="nav-text">Basic functions of BioT2Ex in R</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#add-Description"><span class="nav-number">3.3.1.</span> <span class="nav-text">add_Description</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#write-Out"><span class="nav-number">3.3.2.</span> <span class="nav-text">write_Out</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#add-HeadAnnotation"><span class="nav-number">3.3.3.</span> <span class="nav-text">add_HeadAnnotation</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#add-Feature"><span class="nav-number">3.3.4.</span> <span class="nav-text">add_Feature</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#fix-Number"><span class="nav-number">3.3.5.</span> <span class="nav-text">fix_Number</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#showInExcel"><span class="nav-number">3.3.6.</span> <span class="nav-text">showInExcel</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#op"><span class="nav-number">3.3.7.</span> <span class="nav-text">op</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Running-BioT2Ex-in-Python"><span class="nav-number">4.</span> <span class="nav-text">Running BioT2Ex in Python</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Annotate-the-analysis-result-of-MACS"><span class="nav-number">4.1.</span> <span class="nav-text">Annotate the analysis result of MACS</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#BioT2Ex-m-macs"><span class="nav-number">4.1.1.</span> <span class="nav-text">BioT2Ex -m macs</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Annotate-the-analysis-result-of-MutSigCV"><span class="nav-number">4.2.</span> <span class="nav-text">Annotate the analysis result of MutSigCV</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#BioT2Ex-m-mutsigcv"><span class="nav-number">4.2.1.</span> <span class="nav-text">BioT2Ex -m mutsigcv</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Annotate-the-analysis-result-of-RSEM"><span class="nav-number">4.3.</span> <span class="nav-text">Annotate the analysis result of RSEM</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#BioT2Ex-m-rsem"><span class="nav-number">4.3.1.</span> <span class="nav-text">BioT2Ex -m rsem</span></a></li></ol></li></ol></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">BioT2Ex Group</p>
  <div class="site-description" itemprop="description">BioT2Ex project provides a package or toolkit to conveniently render the tabular txt file into well-anotated Excel files</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives">
          <span class="site-state-item-count">1</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
  </nav>
</div>
  <div class="sidebar-button motion-element"><i class="fa fa-comment"></i>
    Chat
  </a>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2022</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">BioT2Ex Group</span>
</div>

<div class="powered-by">
<i class="fa fa-user-md"></i><span id="busuanzi_container_site_uv">
  Site visits:<span id="busuanzi_value_site_uv"></span>
</span>
</div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>




  















  

  

<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.css">

<script>
NexT.utils.loadComments(document.querySelector('#gitalk-container'), () => {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.js', () => {
    var gitalk = new Gitalk({
      clientID    : '993cd792f87aa14aa3dc# GitHub Application Client ID',
      clientSecret: '4d4cc835e87afbf03582b78383ae42ea3d433343# GitHub Application Client Secret',
      repo        : 'BioT2Extalk# Repository name to store issues',
      owner       : 'Lucinda315# GitHub repo owner',
      admin       : ['Lucinda315# GitHub repo owner and collaborators, only these guys can initialize gitHub issues'],
      id          : '0e40d649a205486e76bb625349959c2c',
        language: 'en',
      distractionFreeMode: true
    });
    gitalk.render('gitalk-container');
  }, window.Gitalk);
});
</script>

</body>
</html>
