<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.4.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-minimal.min.css">
  <script src="/lib/pace/pace.min.js"></script>



<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"lucinda315.github.io","root":"/","scheme":"Mist","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta name="description" content="BioT2Ex currently supports 10 bioinformatics tools, listed below. If you would like another to be added, you can submit your proposal on the Contribute page.   RNA-seq data analysis tools ChIP-seq d">
<meta property="og:type" content="website">
<meta property="og:title" content="BioT2Ex Supported-tools">
<meta property="og:url" content="https://lucinda315.github.io/tools/index.html">
<meta property="og:site_name" content="BioT2Ex">
<meta property="og:description" content="BioT2Ex currently supports 10 bioinformatics tools, listed below. If you would like another to be added, you can submit your proposal on the Contribute page.   RNA-seq data analysis tools ChIP-seq d">
<meta property="og:locale" content="en_US">
<meta property="article:published_time" content="2022-03-18T13:55:05.286Z">
<meta property="article:modified_time" content="2022-03-18T13:54:51.748Z">
<meta property="article:author" content="BioT2Ex Group">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://lucinda315.github.io/tools/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : false,
    lang   : 'en'
  };
</script>

  <title>BioT2Ex Supported-tools | BioT2Ex
</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>
    
    <a target="_blank" rel="noopener" href="https://github.com/Lucinda315">
    <img width="149" height="149" 
     style="position: absolute; top: 0; right: 0; border: 0;" 
      src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1">
    </a>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">BioT2Ex</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a>

  </li>
        <li class="menu-item menu-item-tools">

    <a href="/tools/" rel="section"><i class="fa fa-tools fa-fw"></i>Supported tools</a>

  </li>
        <li class="menu-item menu-item-docs">

    <a href="/document/" rel="section"><i class="fa fa-file-alt fa-fw"></i>Docs</a>

  </li>
        <li class="menu-item menu-item-contribute">

    <a href="/contribute/" rel="section"><i class="fa fa-laptop fa-fw"></i>Contribute</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>About</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          
  
  

          <div class="content page posts-expand">
            

    
    
    
    <div class="post-block" lang="en">
      <header class="post-header">

<h1 class="post-title" itemprop="name headline">BioT2Ex Supported-tools
</h1>

<div class="post-meta">
  

</div>

</header>

      
      
      
      <div class="post-body">
          <div class="modal-body">
<div class="alert alert-info">
BioT2Ex currently supports 10 bioinformatics tools, listed below.
If you would like another to be added, you can submit your proposal on the <a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/contribute/" class="alert-link">Contribute page</a>.
</div>
<ul class="list-inline mod-list-list">
<li class="mod-list rna-seq">RNA-seq data analysis tools</li>
<li class="mod-list chip-seq">ChIP-seq data analysis tools</li>
<li class="mod-list wes">WES data analysis tools</li>
</ul>
<p class="text-muted"><em><span class="fa fa-book"></span> &nbsp;Click the tool name to go to the BioT2Ex documentation for that tool.</em></p>
<table class="table table-condensed">
<tr>
<th>Tool Name</th>
<th>Description</th>
<th>Type</th>
</tr>
<tr><td class="mod-list rna-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_edger">edgeR</a>
</td>
<td><small>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/edgeR.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list rna-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_deseq2">DESeq2</a>
</td>
<td><small>Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/DESeq2.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list rna-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_limma">limma</a>
</td>
<td><small>Data analysis, linear models and differential expression for microarray data.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/limma.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list rna-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_clusterprofiler">clusterProfiler</a>
</td>
<td><small>This package supports functional characteristics of both coding and non-coding genomics data for thousands of species with up-to-date gene annotation. It provides a univeral interface for gene functional annotation from a variety of sources and thus can be applied in diverse scenarios. It provides a tidy interface to access, manipulate, and visualize enrichment results to help users achieve efficient data interpretation. Datasets obtained from multiple treatments and time points can be analyzed and compared in a single run, easily revealing functional consensus and differences among distinct conditions.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list rna-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_noiseq">NOISeq</a>
</td>
<td><small>Analysis of RNA-seq expression data or other similar kind of data. Exploratory plots to evualuate saturation, count distribution, expression per chromosome, type of detected features, features length, etc. Differential expression between two experimental conditions with no parametric assumptions.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/NOISeq.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list chip-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_chipseeker">ChIPseeker</a>
</td>
<td><small>This package implements functions to retrieve the nearest genes around the peak, annotate genomic region of the peak, statstical methods for estimate the significance of overlap among ChIP peak data sets, and incorporate GEO database for user to compare the own dataset with those deposited in database. The comparison can be used to infer cooperative regulation and thus can be used to generate hypotheses. Several visualization functions are implemented to summarize the coverage of the peak experiment, average profile and heatmap of peaks binding to TSS regions, genomic annotation, distance to TSS, and overlap of peaks or genes.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list chip-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#render_diffbind">DiffBind</a>
</td>
<td><small>Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.<a target="_blank" rel="noopener" href="http://www.bioconductor.org/packages/release/bioc/html/DiffBind.html">Tool link</a></small>
</td>
<td>R</td>
</tr>
<tr><td class="mod-list chip-seq"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#biot2ex%20-m%20macs">MACS</a>
</td>
<td><small>A novel algorithm, named Model-based Analysis of ChIP-Seq (MACS), for identifying transcript factor binding sites. MACS captures the influence of genome complexity to evaluate the significance of
enriched ChIP regions, and MACS improves the spatial resolution of binding sites through combining the information of both sequencing tag position and orientation. MACS can be easily used for ChIP-Seq data
alone, or with control sample with the increase of specificity.<a target="_blank" rel="noopener" href="https://pypi.org/project/MACS/">Tool link</a></small>
</td>
<td>Python</td>
</tr>
<tr><td class="mod-list wes"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#biot2ex%20-m%20mutsigcv">MutSigCV</a>
</td>
<td><small>MutSigCV significantly identify mutated genes in large-scale cancer genome research facilitated by massively parallel sequencing technology.<a target="_blank" rel="noopener" href="https://www.mathworks.com/products/compiler/matlab-runtime.html">Tool link</a></small>
</td>
<td>Python</td>
</tr>
<tr><td class="mod-list wes"><a target="_blank" rel="noopener" href="http://biot2ex.biomics.info/document/#biot2ex%20-m%20rsem">RSEM</a>
</td>
<td><small>RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data. The RSEM package provides an user-friendly interface, supports threads for parallel computation of the EM algorithm, single-end and paired-end read data, quality scores, variable-length reads and RSPD estimation. In addition, it provides posterior mean and 95% credibility interval estimates for expression levels. For visualization, It can generate BAM and Wiggle files in both transcript-coordinate and genomic-coordinate. Genomic-coordinate files can be visualized by both UCSC Genome browser and Broad Institute’s Integrative Genomics Viewer (IGV). Transcript-coordinate files can be visualized by IGV. RSEM also has its own scripts to generate transcript read depth plots in pdf format. The unique feature of RSEM is, the read depth plots can be stacked, with read depth contributed to unique reads shown in black and contributed to multi-reads shown in red. In addition, models learned from data can also be visualized. Last but not least, RSEM contains a simulator.<a target="_blank" rel="noopener" href="https://github.com/deweylab/RSEM">Tool link</a></small>
</td>
<td>Python</td>
</tr>
</table>









      </div>
      
      
      
    </div>
    

    
    
    


          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">BioT2Ex Group</p>
  <div class="site-description" itemprop="description">BioT2Ex project provides a package or toolkit to conveniently render the tabular txt file into well-anotated Excel files</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives">
          <span class="site-state-item-count">1</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
  </nav>
</div>
  <div class="sidebar-button motion-element"><i class="fa fa-comment"></i>
    Chat
  </a>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2022</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">BioT2Ex Group</span>
</div>

<div class="powered-by">
<i class="fa fa-user-md"></i><span id="busuanzi_container_site_uv">
  Site visits:<span id="busuanzi_value_site_uv"></span>
</span>
</div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>




  
















<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.css">

<script>
NexT.utils.loadComments(document.querySelector('#gitalk-container'), () => {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.js', () => {
    var gitalk = new Gitalk({
      clientID    : '993cd792f87aa14aa3dc',
      clientSecret: '4d4cc835e87afbf03582b78383ae42ea3d433343',
      repo        : 'BioT2Extalk',
      owner       : 'Lucinda315',
      admin       : ['Lucinda315'],
      id          : 'decodeURI',
        language: 'en',
      distractionFreeMode: true
    });
    gitalk.render('gitalk-container');
  }, window.Gitalk);
});
</script>

  

  


</body>
</html>
